src/collections/contact-requests.ts

```diff
@@ -0,0 +1,101 @@
+import { v4 as uuidv4 } from 'uuid';
```

Comment:
The intention with this file is to free it from file concerns.

We call sql and json util files, and _those_ utils take care of any specifics of files. We can now think about our desired application logic, free of any system constraints in this file. Should make it a lot simpler for people to read/reason with.

Reply:
Good call. One of my tasks was to make a jsonutils :) Thanks!